src/main/java/mate/academy/springboot/datajpa/model/Category.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Entity
```

Comment:
valid table name would be `categories` we should add that too since hibernate creates tables for us